core/libp2p/common_objects/peer.hpp

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_PEER_HPP
+#define KAGOME_PEER_HPP
+
+#include "libp2p/common_objects/multiaddress.hpp"
+#include "libp2p/common_objects/multihash.hpp"
+
+namespace libp2p {
+  namespace common {
+    class Peer {
+     public:
+      struct PeerInfo {
+        const Multihash &peer_id;
+        const Multiaddress &peer_address;
```

Comment:
Well, from the second thought, I decided to store values. Initial attempt was to avoid copies when using the same Multi{hash, address} in several places, but the value should be here